src/client/core/utils/dom.js

```diff
@@ -274,6 +274,14 @@ export function isSelectElement (el) {
     return el.tagName && el.tagName.toLowerCase() === 'select';
 }
 
+export function isInputElement (el) {
+    return el.tagName && el.tagName.toLowerCase() === 'input';
```

Comment:
Hammerhead already has [isInputElement](https://github.com/DevExpress/testcafe-hammerhead/blob/master/src/client/utils/dom.js#L407) in domUtils.